test/unit/app/models/editionable_worldwide_organisation_test.rb

```diff
@@ -0,0 +1,89 @@
+require "test_helper"
+
+class EditionableWorldwideOrganisationTest < ActiveSupport::TestCase
+  test "an ambassadorial role is a primary role and not a secondary one" do
+    worldwide_organisation = create(:worldwide_organisation)
```

Comment:
Howdy! Was just looking at this and noticed that these specs use the old factory

Reply:
Oops!  Too much copy and pasting.

I've raised https://github.com/alphagov/whitehall/pull/8692 to use the correct factory.